### PR TITLE
data collect - Save current configuration params

### PR DESCRIPTION
**Pull request by VHolonec**

### CODE REVIEW

**Comment by dNechita** [comment]
@VHolonec are you going to add the rest of the implementation (besides saving users will want to load. Also, there is nothing done for the viewer) in a separate PR or in this one?

**Comment by VHolonec** [comment]
> @VHolonec are you going to add the rest of the implementation (besides saving, users will want to load. Also, there is nothing done for the viewer) in a separate PR or in this one?

For viewer I will add the implementation in separated PR. Also for load configuration is another implementation.

File: examples/data_collect/main.cpp [hunk 1]
```diff
@@ -335,6 +346,15 @@ int main(int argc, char *argv[]) {
         LOG(INFO) << "Configure camera with " << configuration;
     }
 
+    if(saveconfigurationFile) {
```
[the remark this answers is not in the document]
Comment (VHolonec) [reply examples/data_collect/main.cpp:349]: Done.

File: examples/data_collect/main.cpp [hunk 2]
```diff
@@ -99,6 +100,7 @@ int main(int argc, char *argv[]) {
         {"-t", {"--netlinktest", false, "", "", false}},
         {"-st", {"--singlethread", false, "", "", false}},
         {"-ic", {"--ic", false, "", "", false}},
+         {"-scf", {"--scf", false, "", "", false}},
```
[the remark this answers is not in the document]
Comment (VHolonec) [reply examples/data_collect/main.cpp]: Done.

**Review by dNechita — APPROVED** [review]

**Pull request merged**